week-6-e

pdbhatt opened the pull request.

pdbhatt committed:
Create week-6-e-Perception_Aware_MPC_Paper_review.md

1997Theo commented on bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md:
```diff
@@ -0,0 +1,107 @@
+# Paper review-Perception Aware Model Predictive Control for Quadrotors 
```
Overall, The notes effectively cover key aspects of Perception Aware Model Predictive Control (PAMPC) including its importance, methodology, problem formulation, and conclusions, providing a good overview of the subject. It is also well structured, have a good label in each equations, making the information accessible and logically organized. The main idea of the notes is also clear, giving a holistic description of how MPC integrates with perception for controlling quadrotors.

Some areas to improve in order to make this notes even better : 
1. The writer can include diagrams or figures which could enhance understanding, especially where complex concepts or equations are discussed. Maybe the writer can add visual aids to illustrate the relationships between perception, control, and quadrotor dynamics.
2. Some equations and terms could use more detailed explanations or annotations to aid readers unfamiliar with the specific notation or concepts used in advanced control systems.

pdbhatt replied:
Thank you for your feedback! I wanted to ask if you were having any difficulty in viewing the equations as the GitHub preview showed me some notations that were not clearly translated from code to text/number?

1997Theo replied:
Actually what i did is copying all your code into my VS studio code, and read it there. So far i can see all the equation clearly through my notebook.

1997Theo commented on bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md:
```diff
@@ -0,0 +1,107 @@
+
+Authors - Davide Falanga, Philipp Foehn, Peng Lu, and Davide Scaramuzza
+
+## Why is this important?
```
This section mention clearly about the importance of the proposed algorithm, although it could be better if the writer give more realistic example to emphasize the importance. (Realistic here means the actual case of application to give reader more understanding, especially those who are unfamiliar with autonomous vehicle)

1997Theo commented on bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md:
```diff
@@ -0,0 +1,107 @@
+
+As we are advancing into a world of autonomous control of vehicles, there have been some key developments that have accelerated the research in the domain. One of these, is the Perception Aware Model Predictive Control (PAMPC) for Quadrotors, which I am going to review in this document. The focus of this review is to summarize the Model Predictive Control (MPC) pipeline used along with its cost f ...
+
+## Introduction
```
This part clearly states the relevance and emerging importance of integrating perception with control. Although it would be better if the writer can explain what does perception means in this context to give reader the right understanding. It will also be helpful if the writer can could include a brief overview or examples of previous methods and their limitations to contrast why PAMPC is beneficial.

1997Theo commented on bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md:
```diff
@@ -0,0 +1,107 @@
+
+Previously, perception and control action were treated as two separate problems in controls. To solve this problem, the authors of the paper designed a Model Predictive Control that optimizes the control input for trajectory optimization and simultaneously optimizing the perception of the sensing algorithms to maximize the vision needed to execute a control action. An example of this can be the p ...
+
+## Methodology
```
This part has a good flow and labelling in all the equation. It shows detailed mathematical representation of how perception is integrated into the MPC framework. It would be really good if the writer can give step-by-step explanations or a flowchart describing how each component interacts within the system.

1997Theo commented on bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md:
```diff
@@ -0,0 +1,107 @@
+where $f_{x}, f_{y}$ are the focal lengths for pixel rows and columns, respectively.
+Later, these equations can be used to express the projection velocity as a function of the quadrotor state and input vectors as we are interested in reducing the velocity of its projection onto the image plane. 
+
+## Problem Formulation
```
This part provides a clear formulation of the optimization problem including objective functions and constraints. It would be really good if the writer can explain how these formulations enhance the quadrotor's ability to navigate or respond to environmental variables.

1997Theo commented on bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md:
```diff
@@ -0,0 +1,107 @@
+
+The figure above shows the quadrotors maintaining their velocity at the same time maximizing their vision space around objects.
+
+## References
```
The reference is clear since it mentioned the paper the writer used along with the link

1997Theo commented on bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md:
```diff
@@ -0,0 +1,107 @@
+
+where $c_{\min }, c_{\max }, \Omega_{\max }, v_{\max } \in \mathcal{R}_{+} \cdot$
+
+## Conclusion
```
The writer clearly summarizes the effectiveness of the PAMPC approach and mentions real-world application. The conclusion will be more robust if the writer can also discuss potential limitations, areas for future research, and comparison against traditional MPC approaches.

pdbhatt committed:
Update week-6-emd

This commit includes the response to the peer reviews and includes the changes made.
